.github/actions/docker-build-image/action.yml

```diff
@@ -209,3 +209,16 @@ runs:
         labels: ${{ steps.meta.outputs.labels }}
         push: true
         provenance: false
+
+    - name: Print tags
+      if: env.PUSH_IMAGE == 'true'
+      shell: bash
+      run: |
+        {
+          echo "### :rocket: Docker image built: \'${{ env.IMAGE_NAME }}\'"
+          echo "| Registry            | Image URL |"
+          echo "| ------------------- | ----- |"
+          echo "| Quay                | [quay.io/alfresco/${{ env.IMAGE_NAME }}](quay.io/alfresco/${{ env.IMAGE_NAME }}) |"
+          echo "| GHCR                | [ghcr.io/alfresco/${{ env.IMAGE_NAME }}](ghcr.io/alfresco/${{ env.IMAGE_NAME }}) |"
+          echo "| AWS ECR             | [${{ steps.login-ecr.outputs.registry }}/${{ env.IMAGE_NAME }}](${{ steps.login-ecr.outputs.registry }}/${{ env.IMAGE_NAME }}) |"
```

Comment:
Ok I saw in your intermediate commits that you wanted to ease up copy/paste, and it's not working nicely with tables indeed, so here's another suggestion:

```suggestion
          echo "### :rocket: Docker image built:"
          echo "Quay:"
          echo "```\nquay.io/alfresco/${{ env.IMAGE_NAME }}\n```
          echo "GHCR:"
          echo "```\ghcr.io/alfresco/${{ env.IMAGE_NAME }}\n```
          echo "AWS ECR:"
          echo "```\n${{ steps.login-ecr.outputs.registry }}/${{ env.IMAGE_NAME }}\n```
```
If line returns are working as expected, this should give:

### :rocket: Docker image built
Quay:
```
quay.io/alfresco/alfresco-identity-adapter-service:feature_AAE-23198-action-test
```
GHCR:
```
ghcr.io/alfresco/alfresco-identity-adapter-service:feature_AAE-23198-action-test
```
AWS ECR:
```
645890235518.dkr.ecr.us-east-1.amazonaws.com/alfresco-identity-adapter-service:feature_AAE-23198-action-test
```